<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="admin-PC" ignored="3" name="Default test(failed)" tests="1" failures="1" timestamp="04 May 2019 06:41:56 GMT" time="0.079" errors="0">
  <testcase name="TestFour" time="0.03" classname="com.testng.ex.ExecuiteOnlyFailTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.testng.ex.ExecuiteOnlyFailTest.TestFour(ExecuiteOnlyFailTest.java:32)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- TestFour -->
  <testcase name="TestTwo" time="0.0" classname="com.testng.ex.ExecuiteOnlyFailTest">
    <ignored/>
  </testcase> <!-- TestTwo -->
  <testcase name="TestThree" time="0.0" classname="com.testng.ex.ExecuiteOnlyFailTest">
    <ignored/>
  </testcase> <!-- TestThree -->
  <testcase name="TestOne" time="0.0" classname="com.testng.ex.ExecuiteOnlyFailTest">
    <ignored/>
  </testcase> <!-- TestOne -->
</testsuite> <!-- Default test(failed) -->
